### DISCUSSION TITLE
Message: Illuminate\Database\QueryException

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#7201] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Updating Firefly iii to 6.01. and 6.0.2 also, causes an error with no discernible effects.

`composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii-updated <version>`

Despite the message, Firefly works fine as far as I can verify.

![20230311-0912](https://user-images.githubusercontent.com/58190287/224473222-1b6e4af1-ba3e-4e69-9483-ab7f58dd7d42.jpg)

### Debug information

   Illuminate\Database\QueryException

  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for db failed: Name or service not known (Connection: mysql, SQL: delete from `configuration` where `name` = upgrade_security_message)

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:760
    756▕         // If an exception occurs when attempting to run a query, we'll format the error
    757▕         // message to include the bindings with SQL, which will make this exception a
    758▕         // lot more helpful to the developer instead of just the database's errors.
    759▕         catch (Exception $e) {
  ➜ 760▕             throw new QueryException(
    761▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    762▕             );
    763▕         }
    764▕     }

      +17 vendor frames

  18  app/Support/FireflyConfig.php:49
      Illuminate\Database\Eloquent\Builder::forceDelete()

  19  app/Console/Commands/VerifySecurityAlerts.php:59
      FireflyIII\Support\FireflyConfig::delete()

Script @php artisan firefly-iii:verify-security-alerts handling the post-install-cmd event returned with error code 1

**Comment by JC5** [comment]
It seems to lose the connection to the database and errors out. That's not a bug per se, but perhaps a configuration issue. Is your database reporting issues?

**Comment by ghost** [comment]
I haven't noticed any errors so far.

I also thought that with this first SSH update entry, no database is called, since the `.env` file is not in the new directory yet, and only the new version is installed.

**Comment by JC5** [comment]
Good point, I hadn't actually thought of that. I'll check it out!

**Comment by ghost** [comment]
The update to version 6.0.3 works without an error message again 👍

**Reply by JC5** [reply]
Thanks for the quick feedback, I'm glad to hear it!
